doc/whats_new/v0.21.rst

```diff
@@ -47,7 +47,15 @@ Support for Python 3.4 and below has been officially dropped.
   algoritm related to :class:`cluster.DBSCAN`, that has hyperparameters easier
   to set and that scales better, by :user:`Shane <espg>` and
   :user:`Adrin Jalali <adrinjalali>`.
-  
+
+:mod:`sklearn.semi_supervised`
+..............................
+
+- |MajorFeature| Added :class:`semi_supervised.SelfTrainingClassifier`, a 
+  meta-classifier that allows any supervised classifier to function as a 
+  semi_supervised classifier that can learn from unlabeled data. :issue:`1243`
+  by `Oliver Rausch`_ and `Patrice Becker`_.
```

Comment:
to function -> to behave ? (I'm not native speaker)

The issue number should be that of your PR #11682

Reply:
I'm a native speaker and IMO its fine, but if you'd still like me to change it to  "classifier to behave like a semi_supervised classifier" I can.

Reply:
No problem I trust you on this, thanks

Reply:
Are you sure that it should be the number of the PR? When I visit [the current what's new](http://scikit-learn.org/dev/whats_new.html) and click the links, they all link to issues.

Reply:
Yes that's confusing but it should be the PR number. If you actually click on the links, you'll see that they end up on PRs, not issues